lib/supabase/client.ts

```
@@ -13,47 +13,63 @@ const supabaseStub: any = {
+      data: { url: null },
+      error: { message: 'Supabase not configured' },
+    }),
+    signOut: async () => ({ error: null }),
   },
   from: (_table: string) => ({
     select: () => ({
       eq: () => ({
-        then: async (cb?: any) => { cb?.({ data: [], error: { message: 'Supabase not configured' } }); return { data: [], error: { message: 'Supabase not configured' } } },
-        single: async () => ({ data: null, error: { message: 'Supabase not configured' } })
-      })
+        then: async (cb?: any) => {
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)